.evg.yml

```diff
@@ -102,9 +103,124 @@ tasks:
     commands:
       - func: "generate sbom"
       - func: "upload sbom"
+      - func: "push SBOM Lite to Silk"
       - func: "scan sbom"
 
+  - name: ssdlc-artifacts
+    run_on: ubuntu2204-small
+    depends_on:
+      - name: sbom
+        variant: code-quality-and-correctness
+    exec_timeout_secs: 300 # 5m
+    commands:
+      - func: "pull augmented SBOM from Silk"
```

Comment:
Let's move this between `push SBOM Lite to Silk` and `scan sbom`. This way we will be able to compare the vulnerabilities reported by Silk and those reported by Grype.